lib/mayaUsd/ufe/Utils.cpp

```diff
@@ -600,6 +600,9 @@ Ufe::Attribute::Type usdTypeToUfe(const PXR_NS::SdrShaderPropertyConstPtr& shade
                 { PXR_NS::SdrPropertyTypes->String, PXR_NS::SdfValueTypeNames->String },
                 { PXR_NS::SdrPropertyTypes->Float, PXR_NS::SdfValueTypeNames->Float },
                 { PXR_NS::SdrPropertyTypes->Color, PXR_NS::SdfValueTypeNames->Color3f },
+#if defined(USD_HAS_COLOR4_SDR_SUPPORT)
+                { PXR_NS::SdrPropertyTypes->Color4, PXR_NS::SdfValueTypeNames->Color4f },
+#endif
```

Comment:
Without that patched USD with Color4 support, a MaterialX color4 will look and act as a Float4.